pysatSpaceWeather/tests/test_methods_general.py

```diff
@@ -40,7 +40,6 @@ def test_preprocess(self):
         general.preprocess(self.testInst)
 
         # Test the output
-        for var in self.testInst.meta.keys():
-            assert np.isnan(
-                self.testInst.meta[var, self.testInst.meta.labels.fill_val])
+        assert np.isnan(
+            self.testInst.meta[var, self.testInst.meta.labels.fill_val])
```

Comment:
Since some of the fill vals in the test instrument are set to -1, not all can be NaNs.  Testing for the one variable we change so that we know should be a NaN.